code/obj/item/storage/food.dm

```diff
@@ -93,7 +93,7 @@
 /obj/item/storage/box/ic_cones
 	name = "ice cream cone box"
 	icon_state = "ic_cones"
-	desc = "Ice cream cones stored in a handy box."
+	desc = "A box full of delicious ice cream cones, a brittle pastry in a convenient hollow cone shape. When held, the cone provides an insulative, absorbent, and edible layer between the holder's hands and the ice cream (with a temperature difference of about 30C), allowing the ice cream to be eaten in comfort without worry of accelerated melting, or sticky hands. Because of this ease of use, as well as the precarious position that the often overfilled cones hold the ice cream in, ice cream stored in this way is often eaten without the help of utensils, such as spoons. The edibility of the cone solves logistical issues with discarded packaging, and provides a new texture to the classic treat that many eaters find quite delightful. All in all, if you're looking to serve some ice cream, the humble ice cream cone is the premier choice for carrying your ice cream!"
```

Comment:
it is funny! But it would be more funny ifit follows the code guidelines and is not in one big line...
https://hackmd.io/@goonstation/docs/%2F%40goonstation%2Fcode#Very-long-lines
(it is a joke! XD - But would be better, i think)